modules/sov-ibc-transfer/src/context.rs

```diff
@@ -148,34 +146,27 @@ where
         coin: &PrefixedCoin,
         memo: &Memo,
     ) -> Result<(), TokenTransferError> {
-        let token_id_buf = general_purpose::STANDARD_NO_PAD
-            .decode(memo.as_ref())
-            .map_err(|e| TokenTransferError::Other(format!("Failed to decode memo: {e}")))?;
-
-        let token_id = BorshDeserialize::deserialize(&mut token_id_buf.as_slice())
-            .map_err(|e| TokenTransferError::Other(format!("Failed to decode memo: {e}")))?;
+        if !memo.as_ref().is_empty() {
+            return Err(TokenTransferError::Other(
+                "Memo must be empty when burning tokens".to_string(),
+            ));
+        }
 
-        let expected_token_id = {
+        let minted_token_id = {
```

Comment:
if the parentheses here are not really needed, we can remove them.

Reply:
[b927153](https://github.com/informalsystems/sovereign-ibc/pull/133/commits/b9271531b1b259510603b1c0048725a82dceea78)